### ISSUE TITLE
Handling of monorepos

### ISSUE THREAD

**Issue by jtpio** [jupyter-server/jupyter_releaser#48]
Using the Jupyter Releaser with a monorepo, it looks like the packages are retrieved with glob patterns and might be returning too many results:

![image](https://user-images.githubusercontent.com/591645/123667988-f2e92300-d83a-11eb-845f-51e87d696cc7.png)

Probably the logic is somewhere here:

 https://github.com/jupyter-server/jupyter_releaser/blob/8285f2d33836d344492f1c08b31f33648edee849/jupyter_releaser/npm.py#L154

_Originally posted by @jtpio in https://github.com/jupyter-server/jupyter_releaser/issues/46#issuecomment-869809372_

**Comment by jtpio** [comment]
For reference, this is being tested against https://github.com/jtpio/jupyterlite, but probably applies to similar repos.

Which currently has the following structure (summary):

```bash
app/package.json
app/lab/package.json
app/retro/package.json
buildutils/package.json
py/jupyterlite/pyproject.toml
py/jupyterlite/setup.py
pyproject.toml (just a meta file)
package.json
packages/*
```

And the following `workspace` configuration:

```json
"workspaces": {
  "packages": [
    "app",
    "app/*",
    "buildutils",
    "packages/*"
  ]
},
```

Also in this case the Python package is not at the top-level (and there might be multiple packages at some point in the future)

**Comment by blink1073** [comment]
This should be finally fixed by #56

**Comment by jtpio** [comment]
Yes just tested the draft changelog workflow which seems to go through now (and was failing before): https://github.com/jtpio/jupyterlite/pull/221
